README.md

```diff
@@ -1,2 +1,41 @@
 # ecs-task-metadata-exporter
-A Prometheus exporter for monitornig ECS containers using the ECS task metadata endpoint.
+
+A Prometheus exporter for monitoring ECS containers using the ECS task metadata endpoint. `ecs-task-metadata-exporter` can be run as a sidecar by including it as an additional container in any task definition for deployment onto ECS. It targets tasks run on EC2 instances running ECS agent version `>= v1.21.0` or tasks running on Fargate with platform version `>= 1.3.0`. Windows services are not supported.
+
+## Metrics
+
+As of now, the set of metrics is fairly minimal. Once the system is validated as functioning, more will be added, including network and disk usage metrics.
+For each container, the following are exported, distinguished by the `ContainerName` label.
+- `ecs_container_mem_usage_bytes`: The current memory in use.
+- `ecs_container_mem_max_usage_bytes`: The maximum memory the container has had in use at one time since creation.
+- `ecs_container_mem_limit_bytes`: The maximum memory the container can use, as per the task definition and container runtime.
+- `ecs_container_cpu_usage`: A number from 0 to 1 which represents the ratio of CPU time used by this container compared to the whole host system in a short interval before your request.
+
+In addition, the following metrics about the entire task are exported (no `ContainerName` label is applied):
+- `ecs_container_exporter_up`: 1.0 if no errors were encountered during the the scrape, and 0.0 otherwise. If it returns 0.0, any metrics that were able to be constructed will still be exported.
```

Comment:
Not really understanding this naming. How about `ecs_container_metric_collection_success` or something?

Reply:
I pulled `_up` from https://prometheus.io/docs/instrumenting/writing_exporters/#failed-scrapes